helix-core/src/main/java/org/apache/helix/model/InstanceConfig.java

```diff
@@ -253,7 +254,8 @@ public boolean containsTag(String tag) {
    * @return true if enabled, false if disabled
    */
   public boolean getInstanceEnabled() {
-    return _record.getBooleanField(InstanceConfigProperty.HELIX_ENABLED.toString(), true);
+    return _record.getBooleanField(InstanceConfigProperty.HELIX_ENABLED.toString(),
```

Comment:
NIT: since you change the line, would be good to have name() for toString() as overall replacement for all the code when you change it.